features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/Operations/Operations.jsx

```diff
@@ -204,99 +221,124 @@ class Operations extends React.Component {
         }
         const { classes } = this.props;
         return (
-            <Grid container direction='row' spacing={3} justify='flex-start' alignItems='flex-start'>
-                <Grid item md={12}>
-                    <APIRateLimiting
-                        operationRateLimits={apiPolicies}
-                        api={api}
-                        value={apiThrottlingPolicy}
-                        onChange={this.handleApiThrottlePolicy}
-                    />
-                </Grid>
-                <div className={classes.root}>
-                    <div className={classes.titleWrapper}>
-                        <Typography variant='h4' align='left' className={classes.mainTitle}>
+            <React.Fragment>
+                <Box pb={3}>
+                    <Typography variant='h5'>
+                        <FormattedMessage
+                            id='Apis.Details.Operations.Operations.operation'
+                            defaultMessage='Operations'
+                        />
+                    </Typography>
+                </Box>
+
+                <Grid container spacing={2}>
+                    <Grid item md={12}>
+                        <APIRateLimiting
+                            operationRateLimits={apiPolicies}
+                            api={api}
+                            value={apiThrottlingPolicy}
+                            onChange={this.handleApiThrottlePolicy}
+                        />
+                    </Grid>
+                    <Grid item md={2}>
+                        <Box mt={4} pb={2}>
+                            <div className={classes.searchWrapper}>
+                                <TextField
+                                    id='outlined-full-width'
+                                    label='Operation'
+                                    placeholder='Filter Operations'
+                                    onChange={e => this.filterOperations(e, api.operations)}
+                                    fullWidth
+                                    variant='outlined'
+                                    InputLabelProps={{
+                                        shrink: true,
+                                    }}
+                                />
+                            </div>
+                        </Box>
+                    </Grid>
+                    <Grid item md={12}>
+                        <Table>
+                            <TableRow>
+                                <TableCell>
+                                    <Typography variant='subtitle2'>
+                                        <FormattedMessage
+                                            id='Apis.Details.Resources.Resource.Operation'
```

Comment:
Shouldn't this be changed to "Apis.Details.Operations.Operation.Operations"?

Reply:
yes.Missed it. Thanks!